### ISSUE TITLE
Multiple slashes in path doesn't throw error on windows.

### ISSUE THREAD

**Issue by bsouthga** [benmosher/eslint-plugin-import#557]
A "nice" difference between `*nix` and windows systems is that unix treats duplicate slashes in paths `/User/ben////my////path///` As one slash `/User/ben/my/path/`. 

Windows however does not. This means that on windows, the module loader will be unable to find paths that are okeyed by the linter, for example:

``` javascript
// fine on unix, bad on windows...
import workflow from './/workflow';
```

Is there a way we could add a platform specific check? or maybe just disallow duplicate slashes, as they will be removed automatically on unix?

**Comment by benmosher** [comment]
Yeah, sounds like a good thing to do, in addition to checking that the actual filesystem case matches.

**Comment by danny-andrews** [comment]
Ran into this issue today. Maybe we could name the rule `no-multiple-slashes`. Either that or make a general rule called `no-extraneous-slashes` that could optionally check for trailing slashes, e.g. `import workflow from './workflow/';`

**Comment by danny-andrews** [comment]
As a workaround, you can use the `no-restricted-imports` rule with a `**//` pattern. Example config:
```yaml
rules:
  no-restricted-imports:
    - error
    - patterns: ['**//']
```

**Comment by ljharb** [comment]
I'm going to close this as a duplicate of #417, since #912's `no-useless-path-segments` rule should presumably cover it.

**Comment by danny-andrews** [comment]
@ljharb I think you mean #471, not #417 ;)

**Comment by ljharb** [comment]
Whoops, thanks. Corrected (i'll delete your correction and my reply so there's no more link to 417)
